app/actions/remote/user.ts

```diff
@@ -350,29 +350,30 @@ export const updateUsersNoLongerVisible = async (serverUrl: string): Promise<{er
         return {error: `${serverUrl} database not found`};
     }
 
+    const models: Model[] = [];
     try {
         const knownUsers = new Set(await client.getKnownUsers());
```

Comment:
getKnownUsers will be returning an array right, can it also return other type of value?

Reply:
Get known users will always return an array. If it fails, it will throw an exception, captured by the try/catch block.